### PR TITLE
initial project setup

### PR DESCRIPTION
**Pull request by RitikJaiswal75**
Initial repository setup
This initializes a wrangler project
Contains all the packages required for the project
Contains a demo test.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "ngrok": "^4.3.3",
+    "pre-commit": "^1.2.2",
+    "prettier": "^2.8.1",
+    "ts-jest": "^29.0.5",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.9.4",
+    "wrangler": "^2.5.0"
+  },
+  "dependencies": {
+    "@types/node-fetch": "^2.6.2",
```
Comment (prakashchoudhary07) [inline package.json:36]: Types in dependencies 🤔

**Review by prakashchoudhary07 — CHANGES_REQUESTED** [review]

**Review by yesyash — APPROVED** [review]

File: .github/workflows/pre-commit-test.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    branches: [main, develop]
+jobs:
+  Lint-Check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm install
+      - run: npm run lint-check
+      - run: npm run format-check
+      - run: npm test
```
Comment (rohan09-raj) [inline .github/workflows/pre-commit-test.yaml:13]: Are we not using `yarn` here?

**Review by rohan09-raj — CHANGES_REQUESTED** [review]

File: .github/workflows/pre-commit-test.yaml (hunk 2 above)
Comment (RitikJaiswal75) [reply .github/workflows/pre-commit-test.yaml:13]: NO we are going with NPM here since no specifications were mentioned to use yarn.

Comment (rohan09-raj) [reply .github/workflows/pre-commit-test.yaml:13]: For what, the specifications are not mentioned.
If it's working with Cloudflare Workers, I can see they have command used in there docs

Comment (RitikJaiswal75) [reply .github/workflows/pre-commit-test.yaml:13]: By specifications I mean requirements. It wasn't mentioned as requirement to use YARN

Comment (RitikJaiswal75) [reply .github/workflows/pre-commit-test.yaml:13]: Any specific reason to use yarn specifically?

Comment (rohan09-raj) [reply .github/workflows/pre-commit-test.yaml:13]: `yarn` is comparatively better than `npm` in terms of speed and security. Reason being it parallely installs the dependencies while npm installs it `sequentially`. It also has some more advantages too.

**Review by rohan09-raj — APPROVED** [review]

**Review by prakashchoudhary07 — APPROVED** [review]

**Pull request merged**